pkg/operator/defragcontroller/defragcontroller.go

```diff
@@ -44,7 +44,7 @@ const (
 // observations the controller will perform rolling defragmentation of each etcd member in the cluster.
 type DefragController struct {
 	operatorClient       v1helpers.OperatorClient
-	etcdClient           etcdcli.EtcdClient
+	etcdClient           etcdcli.AllMemberLister
```

Comment:
```suggestion
	memberLister           etcdcli.AllMemberLister
	
	defragClient              Defragment
	statusClient               Status
```

that's freestyle, hope this gets the idea across, no idea if that compiles